modules/virt-vm-storage-volume-types.adoc

```diff
@@ -55,6 +54,14 @@ require persistent storage.
 Only RAW and QCOW2 formats are supported disk types for the container
 image registry. QCOW2 is recommended for reduced image size.
 
+[NOTE]
+====
+A containerDisk volume is ephemeral. It is discarded when
+the virtual machine is stopped, restarted, or deleted. A containerDisk
```

Comment:
```suggestion
the virtual machine is stopped, restarted, or deleted. A container disk
```

Reply:
Went the opposite and ensured these were all referenced as "`containerDisk` volume" so that there is clear distinction between these ephemeral types and container disks, which can also be imported into a PVC and used for persistent storage.